frontend/src/queries/permissions.ts

```diff
@@ -1,18 +1,15 @@
-import {  habitatProxy } from "@/constants";
-import type { OAuthSession } from "@atproto/oauth-client-browser";
+import type { AuthManager } from "@/auth";
 import { queryOptions } from "@tanstack/react-query";
 
-export function listPermissions(session?: OAuthSession) {
+export function listPermissions(authManager: AuthManager) {
   return queryOptions({
     queryKey: ["permissions"],
     queryFn: async () => {
-      const response = await session?.fetchHandler(`/xrpc/com.habitat.listPermissions`, {
-        headers: {
-          'atproto-proxy': habitatProxy,
-        }
-      })
+      const response = await authManager?.fetch(
+        `/xrpc/com.habitat.listPermissions`,
+      );
```

Comment:
this change basically describes what this PR is doing right?
--> we no longer need to atproto-proxy and send along tokens via the pds because habitat is now an oauth provider?